\section{Introduction}
Sequencing and \gls{SNP} chip genotyping are both able to provide genotype information on individuals from a population. They do so at different costs, at different levels of quality, at different density and with different downstream requirements on computational and human resources. In this chapter I investigate these aspects. In addition to the \gls{SNP} array density it is also important, whether a population specific \gls{SNP} array and a population specific reference panel is used when imputing to achieve a greater SNP density, which allows for fine mapping. In chapter \ref{chap:reference_panel} I show that a continent specific array outperforms a 1000G reference panel in terms of imputation accuracy. In this chapter I only focus on identifying the cheapest study design. I do so by comparison of sequence data generated on the Illumina HiSeq2000 platform and SNP array data generated on the Illumina HumanOmni2.5-4 (quad) and -8 (octo) platforms. Specifically I look at the ability to call rare and common variants at lower coverage (figure \ref{fig:downsampling_SNP_count}), the correlation with SNP array data at different coverages (figure \ref{tab:downsampling_omni_intersection}).