%\chapter*{Introduction}
%\chapter{Exploratory analysis of genetic studies in Africa}

\chapter{Introduction}

There is a greater genetic diversity in humans in sub Saharan Africa than anywhere else on the planet.\cite{Bowcock1994}\cite{Jorde1995}\cite{Tishkoff08031996}\cite{Jorde2000}\cite{Stephens2001}\cite{Tishkoff2002}\cite{Stringer2003}\cite{Tishkoff2004}\cite{HapMap2005}\cite{Ramachandran01112005}\cite{Tishkoff22052009}\cite{1000G2010}\cite{1000G2012}\cite{Gurdasani2015}
Due to the recent African origin of modern humans the African continent hosts the largest number of unique variants and thus the best possibility of discovering rare variants affecting common diseases and complex traits.
Despite these facts most populations from which 100 samples or more have been characterized by \gls{WGS} are Caucasian/European, the majority of current commercial genotype chip arrays are designed based on Caucasian/European populations, few \glspl{GWAS} have been conducted in African populations and African populations constitute a relatively small proportion of miscellaneous global sequencing efforts such as 1000G.\cite{1000G2012}
Therefore we are interested in studying genetics in Africa.

\section{Cost effective study design in an African population}
Sequencing and \gls{SNP} chip genotyping are both able to provide genotype information on individuals from a population. They do so at different costs, at different levels of quality, at different density and with different downstream requirements on computational and human resources. Sequencing can be carried out to different depths of coverage; i.e. each base pair position of the genome can be read a certain number of times. The price of sequencing excluding library preparation costs is somewhat linearly proportional to the coverage. In chapter \ref{ch:downsampling} on page \pageref{ch:downsampling} I investigate these aspects and seek to identify the best study design, when taking into account price.

\section{Creation of a continent specific reference panel}
In addition to the \gls{SNP} array density it is also important, whether a population specific \gls{SNP} array and a population specific reference panel is used when imputing to achieve a greater SNP density, which allows for fine mapping. In chapter \ref{ch:reference_panel} on page \pageref{ch:reference_panel} I show that a continent specific array outperforms a 1000G reference panel in terms of imputation accuracy. Having a continent specific haplotype reference panel further increases the value of any SNP array. In chapter \ref{ch:reference_panel} I present preliminary efforts on the generation of such a reference panel. In section \ref{sec:adrp} on \pageref{sec:adrp} I describe the preparation of a reference panel incorporating additional haplotypes from additional populations using improved variant calling, variant filtering, genotype likelihood refinement and phasing. By merging the samples upstream a reference panel of greater quality and better accuracy upon utilisation will be generated.

\section{Preliminary work on the generation of a continent specific SNP array}
Because genotyping platforms designed for \glspl{GWAS} of European populations are not of equal utility in African populations due to sample selection bias in the design of these \gls{SNP} arrays, I will describe the theoretical design a \gls{SNP} array specific to the African continent, which takes into account the greater number of variants and the different \gls{LD} structure in African populations. The design of such a chip would enable researchers in Africa to carry out cost effective \glspl{GWAS}. I present an analysis plan in chapter \ref{ch:chip_design} on page \pageref{ch:chip_design} regarding the design of such a \gls{SNP} array. And I present preliminary work based on 3 African populations in chapter \ref{ch:chip_design}, which lays the foundation for this further work.

\section{Future work}
In addition to the completion of the continent specific reference panel as part of the \gls{ADRP} project I also have other plans for my PhD. Genetics is moving from \gls{GWAS} to functional characterisation of \glspl{SNP}. Therefore I wish to carry out a statistical evaluation the structural effects of \gls{ns} \glspl{SNP} at the \gls{PPI} level. I briefly describe these plans on page \pageref{ch:future}. I also intend to study the introgression of viral \gls{DNA} into human \gls{DNA}. I briefly describe this on page \pageref{ch:future}.

%Because \glspl{NCD} are on the rise as the most common cause of death in \gls{SSA} and because \gls{HIV} and \gls{AIDS} is expected to still be the most common cause of death in \gls{SSA} in 2030 according to the \gls{WHO}\cite{10.1371/journal.pmed.0030442}
%http://www.who.int/healthinfo/global_burden_disease/projections/en/
%my PhD will focus mostly on analysis of DNA from individuals from \gls{SSA}, which are under this disease burden.
%Specifically I will work with samples from a general population cohort study in Uganda\cite{Asiki01022013} of which some have been genotyped on a dense SNP array and others sequenced to low coverage.

%___INTRO 1___

%We are interested in studying genetics in Africa, because the continent has a greater genetic diversity than any other because of the founder effect and the origin of the human species in Africa\cite{1000G2012}\cite{Gurdasani2015}\cite{Tishkoff22052009}\cite{Stringer2003}. Because \glspl{NCD} are on the rise as the most common cause of death in \gls{SSA} and because \gls{HIV} and \gls{AIDS} is expected to still be the most common cause of death in \gls{SSA} in 2030 according to the \gls{WHO}\cite{10.1371/journal.pmed.0030442} % http://www.who.int/healthinfo/global_burden_disease/projections/en/
%my PhD will focus mostly on analysis of DNA from individuals from \gls{SSA}, which are under this disease burden. Specifically I will work with samples from a general population cohort study in Uganda\cite{Asiki01022013} of which some have been genotyped on a dense SNP array and others sequenced to low coverage.

%Because genotyping platforms designed for \glspl{GWAS} of European populations are not of equal utility in African populations due to sample selection bias in the design of these \gls{SNP} arrays, I will seek to design a \gls{SNP} array specific to the African continent as part of my PhD, which takes into account the greater number of variants and the different \gls{LD} structure in African populations. This work will be carried out in collaboration with \acrfull{H3A} and will enables researchers in Africa to carry out cost effective \glspl{GWAS}. I present an analysis plan in section \ref{sec:chip_design} regarding the design of such a \gls{SNP} array. And I present preliminary work based on 3 never before sequenced populations in section \ref{sec:chip_design}.

%Genetics is moving from \gls{GWAS} to functional characterisation of \glspl{SNP}. Therefore I wish to carry out a statistical evaluation the structural effects of \gls{ns} \glspl{SNP} at the \gls{PPI} level. I briefly describe these plans on page \pageref{chap:future}. I also intend to study the introgression of viral DNA into human DNA. I briefly describe this on page \pageref{chap:future}.

%OLD INTRODUCTION

%There is a greater genetic diversity in humans in sub Saharan Africa than anywhere else on the planet.\cite{Bowcock1994}\cite{Jorde1995}\cite{Tishkoff08031996}\cite{Jorde2000}\cite{Stephens2001}\cite{Tishkoff2002}\cite{Tishkoff2004}\cite{HapMap2005}\cite{Ramachandran01112005}\cite{Tishkoff22052009}\cite{1000G2010}\cite{1000G2012}\cite{Gurdasani2015} Due to the recent African origin of modern humans the African continent hosts the largest number of unique variants and thus the best possibility of discovering rare variants affecting common diseases and complex traits. Despite these facts most populations from which 100 samples or more have been characterized by \gls{WGS} are Caucasian/European, the majority of current commercial genotype chip arrays are designed based on Caucasian/European populations, few \glspl{GWAS} have been conducted in African populations and African populations constitute a relatively small proportion of miscellaneous global sequencing efforts such as 1000G.\cite{1000G2012}
%To inform the design of genetic studies in Africa, we genotyped and sequenced samples from three distinct sub Saharan African populations in the East (Baganda), South (Zulu) and North (Ethiopia) and generated a continent-specific reference panel. One of the reasons for designing a haplotype reference panel is that the accuracy of imputation and the accuracy of downstream analyses - e.g. the ability to identify novel association signals in a \gls{GWAS} - improve significantly, which has been shown previously.\cite{Jallow2009}\cite{Gurdasani2015} The presence of African populations in existing reference panels is sparse. To generate an improved reference panel we used low coverage data (4x) from the 3 populations (page \pageref{sec:agv_data_description}) to generate a reference panel merged with phase 1 of 1000G.

%Sequencing and \gls{SNP} chip genotyping are both able to provide genotype information on individuals from a population. They do so at different costs, at different levels of quality, at different density and with different downstream requirements on computational and human resources. In this chapter I investigate these aspects. The ability to sequence also depends on the sequence content and the ability to map sequence reads depends on the uniqueness of the sequence and the presence and length of indels. In addition to the \gls{SNP} array density it is also important, whether a population specific \gls{SNP} array and a population specific reference panel is used when imputing to achieve a greater SNP density, which allows for fine mapping. In section \ref{sec:reference_panel} I show that a continent specific array outperforms a 1000G reference panel in terms of imputation accuracy. In the next section \ref{sec:downsampling} I only focus on identifying the cheapest study design. I do so by comparison of sequence data generated on the Illumina HiSeq2000 platform and SNP array data generated on the Illumina HumanOmni2.5-4 (quad) and -8 (octo) platforms. Specifically I look at the ability to call rare and common variants at lower coverage (figure \ref{fig:downsampling_SNP_count}), the correlation with SNP array data at different coverages (figure \ref{tab:downsampling_omni_intersection}).

%AGVP Introduction

%Globally, human populations show structured genetic diversity as a result of geographical dispersion, selection and drift. Understanding this variation can provide insights into evolutionary processes that shape both human adaptation and variation in disease susceptibility1. Although the Hapmap Project2 and the 1000 Genomes Project3 have greatly enhanced our understanding of genetic variation globally, the characterization of African populations remains limited. Other efforts examining African genetic diversity have been limited by variant density and sample sizes in individual populations4, or have focused on isolated groups, such as hunter gatherers (HG)5, 6, limiting relevance to more widespread populations across Africa.

%The African Genome Variation Project (AGVP) is an international collaboration that expands on these efforts by systematically assessing genetic diversity among 1,481 individuals from 18 ethno-linguistic groups from sub-Saharan Africa (SSA) (Fig. 1 and Supplementary Methods Tables 1 and 2) with the HumanOmni2.5M genotyping array and whole-genome sequences (WGS) from 320 individuals (Supplementary Methods Table 2). Importantly, the AGVP has evolved to help develop local resources for public health and genomic research, including strengthening research capacity, training, and collaboration across the region. We envisage that data from this project will provide a global resource for researchers, as well as facilitate genetic studies in Africa7.

%AGVP Discussion

%The marked haplotype diversity within Africa has important implications for the design of large-scale medical genomics studies across the region, as well as studies of population history and evolution. In this context, the AGVP is a resource that will facilitate a broad range of genomic studies in Africa and globally.

%Although Africa is the most genetically diverse region in the world, we provide evidence for relatively modest differentiation among populations representing the major sub-populations in SSA, consistent with recent population movement and expansion across the region beginning around 5,000 years ago—the Bantu expansion8. Although the history of the Bantu expansion is probably complex, assessments of population admixture can provide new insights. We note historically complex and regionally distinct admixture with multiple HG and Eurasian populations across SSA, including ancient HG and Eurasian ancestry in West and East Africa and more recent complex HG admixture in South Africa. As well as explaining genetic differentiation among modern populations in SSA, these admixture patterns provide genetic evidence for early back-to-Africa migrations, the possible existence of extant HG populations in western Africa—compatible with archaeological evidence15, and patterns of gene flow consistent with the Bantu expansion, including genetic assimilation of populations resident across the region.

%This admixture also has important implications for the assessment of differentiation and positive selection in Africa. Accounting for these elements, we have identified loci under positive selection that are linked with hypertension, malaria, and other pathogens. This provides a proof-of-concept for the ability of geographically widespread genetic data within Africa to identify loci under selection related to diverse environments.

%Our evidence for the broad transferability of genetic association signals and their statistical refinement has important implications for medical genetic research in Africa. Importantly, we highlight that such studies are feasible and can be enabled through the development of more efficient genotype arrays and diverse WGS reference panels for accurate imputation of common variation. In this context, we describe a framework for a new pan-African genotype array that could directly facilitate large-scale genomic studies in Africa.

%A critical next step is the large-scale deep sequencing of multiple and diverse populations across Africa, which should be integrated with ancient DNA data. This would enable us to identify and understand signals of ancient admixture, patterns of historical population movements, and to provide a comprehensive resource for medical genomic studies in Africa.