%\chapter*{Introduction}
\section{Exploratory analysis of genetic studies in Africa}

We are interested in studying genetics in Africa, because the continent has a greater genetic diversity than any other because of the founder effect and the origin of the human species in Africa\cite{1000G2012}\cite{Gurdasani2015}\cite{Tishkoff22052009}\cite{Stringer2003}. Because \glspl{NCD} are on the rise as the most common cause of death in \gls{SSA} and because \gls{HIV} and \gls{AIDS} is expected to still be the most common cause of death in \gls{SSA} in 2030 according to the \gls{WHO}\cite{10.1371/journal.pmed.0030442} % http://www.who.int/healthinfo/global_burden_disease/projections/en/
my PhD will focus mostly on analysis of DNA from individuals from \gls{SSA}, which are under this disease burden. Specifically I will work with samples from a general population cohort study in Uganda\cite{Asiki01022013} of which some have been genotyped on a dense SNP array and others sequenced to low coverage.

Because genotyping platforms designed for \glspl{GWAS} of European populations are not of equal utility in African populations due to sample selection bias in the design of these \gls{SNP} arrays, I will seek to design a \gls{SNP} array specific to the African continent as part of my PhD, which takes into account the different \gls{LD} structure in African populations. This work will be carried out in collaboration with \acrfull{H3A} and will enables researchers in Africa to carry out cost effective \glspl{GWAS}. I present an analysis plan in chapter \ref{chap:chip_design} regarding the design of such a \gls{SNP} array.

Genetics is moving from \gls{GWAS} to functional characterisation of \glspl{SNP}. Therefore I wish to carry out a statistical evaluation the structural effects of \gls{ns} \glspl{SNP} at the \gls{PPI} level. I briefly describe these plans in chapter \ref{chap:future}. I also intend to study the introgression of viral DNA into human DNA. I briefly describe this in chapter \ref{chap:future}.