\section{Conclusion}

The \gls{LD} structure is different between continents. The selection of tag \glspl{SNP}, which can best capture this alternate \gls{LD} pattern on the African continent, is important for increasing the ability to correctly estimate genotype probabilities of vicinal SNPs, which will for example lead to increased power in down-stream \glspl{GWAS}. Here it has been shown, that a theoretical 1M SNP array can capture common variation in a diverse set of African populations (figures \ref{fig:afr5pop} and \ref{fig:f10}). In combination with a continent specific reference panel (page \pageref{ch:reference_panel}) this would probably lead to a continent specific SNP array being comparable in performance to low coverage sequencing (page \pageref{ch:downsampling}).

If more time and additional sequencing data had been available at the time of analysis, then it would have been interesting to compare the imputation accuracy, when using the Omni2.5M SNPs and the 1 million selected tag SNPs as an imputation scaffold. Given the large number of monomorphic SNPs on the Omni2.5M array and the European ascertainment bias, then it is expected, that the 1 million continent specific tag SNPs might perform better in terms of imputation accuracy.
