\section{Results}

\subsection{Imputation accuracy using 1M tagging SNPs}
The coverage for common variants (\gls{MAF} greater than 0.05) across all populations using 1M tagging variants is 0.93 (figure \ref{fig:f10}) suggesting that a 1M density array could potentially capture substantial proportions of variation across Africa. Using the hybrid approach improved the efficiency of tagging (figure \ref{fig:f10}) and using 10 cycles further improved the efficiency over using fewer cycles (data not shown).

\subsection{Comparison of RagTagger with existing methods}
RagTagger uses less than 1GB of memory when selecting tag SNPs from a 2\gls{Mbp} fragment and runs to completion in a few minutes. TAGster\cite{Xu2007} uses more than 10GB of memory (16.5GB) on the same 2\gls{Mbp} fragment and takes longer than a day to run to completion (1.18 days). TAGster fails to run to completion when run across the whole genome and RagTagger finishes in 3 hours using less than 10GB of memory. FESTA (Fragmented Exhaustive Search for TAgSNPs)\cite{Qin15012006} failed to run to completion within 48 hours.