\section{Discussion}

Low coverage data and SNP array data have been shown to both be viable methods for capturing common variation in European populations\cite{Pasaniuc2012}, which increases the power of \glspl{GWAS}. Here we have shown, that the same holds true for African populations (page \pageref{subsec:result_downsampling_sensspec}). SNP array data does not capture novel variants, but these will be fewer as reference panels for imputation improve in terms of their breadth and specificity. Likewise the imputation inaccuracies will be fewer upon development of improved reference panels (page \pageref{ch:reference_panel}) and SNP arrays (page \pageref{ch:chip_design}) specific to the African continent.
As sample sizes grow the step involving refinement of genotype probabilities after variant calling from low coverage data will grow to be very costly. If the accurate capture of singletons and other rare and novel variants is of interest, then the conclusion is to avoid low coverage data and instead sequence to 15x or 30x coverage.
